src/test/java/tests/SearchTest.java

```diff
@@ -0,0 +1,48 @@
+package tests;
+
+import base.AbstractBaseTest;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import pages.HomePage;
+import pages.SpongeBobCollectionPage;
+
+public class SearchTest extends AbstractBaseTest {
+
+    @Test()
+    public void checkSearch() throws InterruptedException {
```

Comment:
as I remember we agreed to add an empty line after the test method

Reply:
corrected